docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -71,6 +74,20 @@ The top bar of Teleport Connect consists of:
 
 Alternatively, you can look for the server in the search bar and press `Enter` to connect to it.
 
+## Opening a local terminal
+
+To open a terminal with a local shell session, either select "Open new terminal" from the additional
+actions menu or use [the keyboard shortcut](#keymap.newTerminalTab).
```

Comment:
This is kind of unconventional, let me know what you think about it. Instead of repeating the shortcut here, I decided to make a link to the relevant part of the config docs which describes the shortcut. However, in the "Connecting to multiple clusters" section of the page we repeat another shortcut verbatim instead of linking.

The shortcut is also visible in the UI when you open the additional actions menu.

Reply:
I like this idea, but if the linter does not allow this then probably we will have to repeat the shortcut.

Reply:
I think, since there's not a lot of information at the link destination, it's worth it to repeat the information here. That way, users don't need to hop around the page if they're only interested in opening a local terminal.